core/domain/fs_domain_test.py

```diff
@@ -65,7 +67,7 @@ def test_delete(self):
         self.fs.delete('abc.png')
         self.assertFalse(self.fs.isfile('abc.png'))
         with self.assertRaisesRegexp(
-            IOError, r'File abc\.png not found'):
+            IOError, re.escape('File abc.png not found')):
```

Comment:
```suggestion
                IOError, re.escape('File abc.png not found')):
```

Reply:
@vojtechjelinek Why is the space necessary, I think the spaces are correct. And even if I add them, there will be lint errors.

Reply:
Ah, hmm, okay in that case please do this
```suggestion
            IOError, re.escape('File abc.png not found')
        ):
```

Reply:
The main problem is that the stuff inside the function is intended to the same level that the stuff inside the with-block.

Reply:
Done.